test/integration/datasources_k8s.bats

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bats
+
+load helper
+
+function setup () {
+  mkdir -p /var/run/secrets/kubernetes.io/serviceaccount
```

Comment:
Would much rather a temporary directory be used here - these tests can be run outside of a container, and I don't want to risk accidentally clobbering something on someone's system.

Reply:
this is where the client is going to get the serviceaccount data needed to access de k8s api, it is not configurable, so I cannot do so much here. Anyway /var/run is a tmpfs (filesystem on memory), so if the teardown is not able to remove it, whenever you reboot these files will be gone

Reply:
Ah. `/var/run` may be `tempfs` on some OSes, but not macOS (where I sometimes run bare integration tests).

We'll leave this like this for now. Hopefully this path becomes configurable at some point...